kafka-ui-react-app/src/components/common/NewTable/LinkTopic.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { CellContext } from '@tanstack/react-table';
+import { NavLink } from 'react-router-dom';
+import { clusterTopicsPath } from 'lib/paths';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const LinkTopic: React.FC<CellContext<any, unknown>> = ({ getValue, row }) => {
+  const value = getValue<string>();
+  const handleClick: React.MouseEventHandler = (e) => e.stopPropagation();
+  return (
+    <NavLink
+      style={{ color: 'blue' }}
+      to={clusterTopicsPath(row.original.name)}
+      title={value}
+      onClick={handleClick}
+    >
+      {value}
+    </NavLink>
+  );
+};
+
```

Comment:
I don't like the approach when we create toipc link component. Let's try to create universal component where we will send `to` as a property